apps/sms/views/shared/js/selection_handler.js

```diff
@@ -72,9 +72,8 @@ SelectionHandler.prototype = {
       return;
     }
     var target = event.target;
-    var value = target.value;
+    var value = target.getAttribute('value');
```

Comment:
I think it might be better if the gaia-checkbox also have the value getter to sync with the native input checkbox attribure.